### PR TITLE
added Versions & Dependencies section to README

### PR DESCRIPTION
**Pull request by oleg-andreyev**
In attempt to address #1505 updated "Dependencies" section in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,14 @@ Contributing
...
 
-Dependencies
+Versions & Dependencies
 ------------
-| Project | Version | Required |
-|---------|---------|----------|
-|[Elasticsearch](https://github.com/elasticsearch/elasticsearch/tree/6.0)|6.0|yes|
+This project tries to follow Elasticsearch in terms of [End of Life](https://www.elastic.co/support/eol) and maintenance since 5.x
+
+| ruflin/elastica                                                                         | ElasticSearch | elasticsearch-php | PHP      |
```
Comment (ruflin) [inline README.md]: Make `elastica` upper case if possible `Elastica`

Comment (oleg-andreyev) [reply README.md]: sure.

**Comment by oleg-andreyev** [comment]
I'll try to figure this out and will update PR as soon as possible

File: README.md (hunk 1 above)
Comment (oleg-andreyev) [reply README.md]: I've updated title to `Elatica`

**Comment by oleg-andreyev** [comment]
@ruflin I'd like to push this forward without putting max version of PHP, because it's quite complicated, but I'll prepare a separate PR as soon as it will be done.

**Comment by ruflin** [comment]
@oleg-andreyev SGTM. Merged.

**Comment by oleg-andreyev** [comment]
@ruflin thanks!

**Pull request merged**